GitUI/UserControls/RevisionGridClasses/DvcsGraph.cs

```diff
@@ -462,6 +472,7 @@ private void RebuildGraph()
             Invalidate(true);
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2002:DoNotLockOnObjectsWithWeakIdentity")]
```

Comment:
:bulb: I'm not convinced this is correct. I would prefer to see it moved to its own pull request and evaluated separately.